source/pkg/states/PlayState.hx

```diff
@@ -3,7 +3,12 @@ package pkg.states;
 import flixel.FlxG;
 import flixel.FlxSprite;
 import flixel.FlxState;
+import flixel.FlxSubState;
+import js.html.SelectElement;
```

Comment:
Do we need this? js stuff stops me from running it in neko

Reply:
Must have auto-imported